librz/core/rop.c

```diff
@@ -149,18 +151,31 @@ static bool rz_rop_print_quiet_mode(const RzCore *core, const RzCoreAsmHit *hit,
 		RzAsmParseParam *param = rz_asm_get_parse_param(core->analysis->reg, aop.type);
 		RzStrBuf *colored_asm = rz_asm_colorize_asm_str(bw_str, core->print, param, asmop->asm_toks);
 		rz_asm_parse_param_free(param);
-		rz_cons_printf(" %s%s;", colored_asm ? rz_strbuf_get(colored_asm) : "", Color_RESET);
+		if (context->ret_val) {
+			char *str_buf = rz_str_newf(" %s%s;", colored_asm ? rz_strbuf_get(colored_asm) : "", Color_RESET);
```

Comment:
use `rz_strbuf_appendf` and seems that you always inject `Color_RESET` if it is not colored. this might be a problem